src/satellite/package.json

```diff
@@ -7,8 +7,8 @@
     "test:watch": "jest --watch",
     "test": "jest -c jest.config.js",
     "coverage": "jest -c jest.config.js --collect-coverage",
-    "eslint": "eslint --config .eslintrc.js \"**/*.js\"",
-    "eslint-fix": "eslint --config .eslintrc.js \"**/*.js\" --fix",
+    "eslint": "eslint --config .eslintrc.js",
+    "eslint-fix": "eslint --config .eslintrc.js --fix",
```

Comment:
ESLint looks at this pattern so it can go to each file, and apply the rules in `.eslintrc.js` on them. I might be wrong, but that was at least how it works the last time I worked with it. I once added this to make eslint work in Telescope https://github.com/Seneca-CDOT/telescope/pull/2532/files